lib/collection/script.js

```diff
@@ -62,6 +63,9 @@ _.assign(Script.prototype, /** @lends Script.prototype */ {
          * @type {string}
          */
         this.type = options.type || 'text/javascript';
+
+        this.packages = options.packages || {};
```

Comment:
Do we need to add a default value if this key is not present? If it's an optional property, I'm guessing... no?

Reply:
@coditva Do we mean to make this completely optional? 
For consistency purposes, I thought it would be best to have this property in all event objects, and rather than leaving it as undefined in the cases where it isn't available, it seemed better to have it as an empty object.

We can change this in such a way that we move this under an 
if (options.packages) to make this completely optional, but it does not seem ideal IMO. Do let me know if this is not the case.

It would be completely optional from a User perspective in either way tho.

Reply:
A whole lot of collections will start having empty `packages` field. I want to avoid it for the collections that don't even use it.

Reply:
makes sense, I'll make it so that this.packages is only set if options.packages exists. Sounds good?